src/panels/lovelace/cards/hui-warn-card.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+
+export class HuiWarnCard extends LitElement {
```

Comment:
What's different about this one compared to `hui-warn-card` ?

Reply:
nothing at all. didn't mean to check that it in. my first thought was to create that, and then thought better of it. thanks